modules/nw-networkpolicy-about.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// networking/configuring-networkpolicy.adoc
+
+[id="nw-networkpolicy-about-{context}"]
+
+= About network policy
+
+In {product-title} {product-version} `NetworkPolicy` is the default mode for the
```

Comment:
Also wondering whether "NetworkPolicy" really needs to be in backticks everywhere. For things like ConfigMap, DeploymentConfig, etc., the guidance is to not use backticks.